frontend/context/StakingProgramProvider.tsx

```diff
@@ -2,21 +2,22 @@ import { useQuery, useQueryClient } from '@tanstack/react-query';
 import { Maybe } from 'graphql/jsutils/Maybe';
 import { createContext, PropsWithChildren, useCallback } from 'react';
 
+import { INITIAL_DEFAULT_STAKING_PROGRAM_IDS } from '@/config/stakingPrograms';
 import { FIVE_SECONDS_INTERVAL } from '@/constants/intervals';
 import { REACT_QUERY_KEYS } from '@/constants/react-query-keys';
 import { StakingProgramId } from '@/enums/StakingProgram';
 import { useServiceId } from '@/hooks/useService';
 import { useServices } from '@/hooks/useServices';
 import { Nullable } from '@/types/Util';
 
-const INITIAL_DEFAULT_STAKING_PROGRAM_ID = StakingProgramId.PearlBeta;
-
 export const StakingProgramContext = createContext<{
   isActiveStakingProgramLoaded: boolean;
+  initialDefaultStakingProgramId: Maybe<StakingProgramId>;
   activeStakingProgramId: Maybe<StakingProgramId>;
 }>({
   isActiveStakingProgramLoaded: false,
   activeStakingProgramId: null,
+  initialDefaultStakingProgramId: null,
```

Comment:
i think this should always be initialized with a value 🤔 
to avoid Maybe type
don't need to fix now though